docs/stacks-guide-rook.md

```diff
@@ -0,0 +1,874 @@
+          - name: yedis
+            port: 6379          # default value
+          - name: ysql
+            port: 5433          # default value
+      # Volume claim template for TServer
+      volumeClaimTemplate:
+        metadata:
+          name: datadir
+        spec:
+          accessModes: [ "ReadWriteOnce" ]
+          resources:
+            requests:
+              storage: 1Gi
+          storageClassName: standard
+EOF
+
+kubectl apply -f yugabyteclass.yaml
+```
+
+## Provision Target Cluster
+
+### Provision GKE Cluster with a Resource Claim
+
+Now that our classes have been created, we need to provision the GKE cluster by
+creating a `KubernetesCluster` claim.
+
+```bash
+cat > k8sclaim.yaml <<EOF
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: yugastore-app
+  labels:
+    app: yugastore
+---
+apiVersion: kubernetes.crossplane.io/v1alpha1
+kind: Provider
+metadata:
+  name: yugastore-k8s-provider
+  labels:
+    app: yugastore
+spec:
+  credentialsSecretRef:
+    name: yugastore-k8s-secret
+    namespace: yugastore-app
+---
+apiVersion: compute.crossplane.io/v1alpha1
+kind: KubernetesCluster
+metadata:
+  name: yugastore-k8s
+  namespace: yugastore-app
+  labels:
+    app: yugastore
+spec:
+  classSelector:
+    matchLabels:
+      app: yugastore
+  writeConnectionSecretToRef:
+    name: yugastore-k8s-secret
+EOF
+
+kubectl apply -f k8sclaim.yaml
+```
+
+You can view the status of the `KubernetesCluster` claim as it waits for the GKE
+cluster to come available:
+
+*Command*
+```bash
+kubectl get kubernetesclusters -n yugastore-app
+```
+*Output*
+```bash
+NAME            STATUS   CLASS-KIND        CLASS-NAME     RESOURCE-KIND   RESOURCE-NAME                       AGE
+yugastore-k8s   Bound    GKEClusterClass   standard-gke   GKECluster      yugastore-app-yugastore-k8s-vdhjq   23m
+```
+
+You can view the status of the `GKECluster` itself as it is created:
+
+*Command*
+```bash
+kubectl get gkeclusters.compute.gcp.crossplane.io yugastore-app-yugastore-k8s-vdhjq
+```
+*Output*
+```bash
+NAME                                STATUS   STATE     CLUSTER-NAME                               ENDPOINT        CLUSTER-CLASS   LOCATION        RECLAIM-POLICY   AGE
+yugastore-app-yugastore-k8s-vdhjq   Bound    RUNNING   gke-20a8dd85-b76c-4041-bc77-36c13ee28e37   35.224.89.145   standard-gke    us-central1-b   Delete           5m41s
+```
+
+Here we have created a namespace `yugastore-app` for our Yugastore namespaced
+resources to use and also create a Kubernetes `Provider` the references the
+secret propagated by the `KubernetesCluster` claim. If you look back at the
+`YugabyteClusterClass` we created above, this `yugastore-k8s-provider` is
+referenced, so once the secret is propagated, the Rook stack will be able to use
+it to provision a `YugabyteCluster`. However, before we get to that, we need to
+deploy the Rook Yugabyte operator into the Kubernetes cluster.
+
+### Install Rook Yugabyte Operator with a `KubernetesApplication`
```

Comment:
comment from above, suggest using just plain text for KubernetesApplication without the inline code markdown for the heading